data/toc.yaml

```diff
@@ -1795,6 +1793,18 @@ Manuals:
           title: OpenTelemetry support
         - path: /build/building/base-images/
           title: Create your own base image
+    - sectiontitle: Build context
+      section:
+        - path: /build/context/
+          title: Overview
+        - path: /build/context/local/
+          title: Local context
+        - path: /build/context/remote/
+          title: Remote context
+        - path: /build/context/empty/
+          title: Empty context
+        - path: /build/context/dockerignore/
+          title: Dockerignore file
```

Comment:
I'm not sure about moving the context page to a section and splitting with multiple pages tbh. I think it should still be under "Building images" like multi-stage, it's not a specific paradigm.

Might be a matter of personal taste but I prefer a single page treating of the same subject: easier to search in, don't need to jump around pages, readability on mobile.

Reply:
Happy to merge it back to a longer page. I'll give it a try.